DEV-10/ElementChecker.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections;
+using static System.Math;
+
+namespace RepeatElementInArray
+{ /// <summary>
+  /// For checking elements on uniqueness.
+  /// </summary>
+  class ElementChecker
+  {
+    const double EPSILON = 10e-2;
+    /// <summary>
+    /// For checking elements on uniqueness.
+    /// </summary>
+    /// <param name="array"> 
+    /// The array which we need to check.
+    /// </param>
+    /// <returns>
+    /// An array just with duplicate elements.
+    /// </returns>
+    public ArrayList CheckElements(double[][] array)
```

Comment:
Bad name, this method not only check elements, its creates new array, single responsibility